### PR TITLE
Add code and release notes for SliceDhv* devices.

### LINKED ISSUE #916: Add Device for Vescent Slice-DHV Piezo Controller
## Expected Behavior
We would like to have a class for the Vescent Slice-DHV piezo controller for hutch python and Typhos. 

## Current Behavior
No device class exists.

### PR DESCRIPTION
**Pull request by slactjohnson**
## Description
Add two sub classes, and a top-level class, for the Slice-DHV piezo driver. This driver will be used in the new LCLS timing system. 

## Motivation and Context
Closes #916 

## How Has This Been Tested?
Tested on a controller in the ASC lab, using a typhos screen generated by this class.

## Where Has This Been Documented?
The code, and docstrings. The PVs are fairly self-explanatory. 

## Screenshots (if appropriate):
image

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]
This looks good to me, one quibble in the release notes

File: docs/source/upcoming_release_notes/916-Add_slice-dhv.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- tjohnson
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/916-Add_slice-dhv.rst:35]: github name here
```suggestion
- slactjohnson
```

File: pcdsdevices/piezo.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    model = Cpt(EpicsSignalRO, ':MODEL', kind='omitted')
+
+    serial = Cpt(EpicsSignalRO, ':SERIAL', kind='omitted')
+
+    ctl_fw = Cpt(EpicsSignalRO, ':CTL_FW', kind='omitted')
+
+    dhv_fw = Cpt(EpicsSignalRO, ':DHV_FW', kind='omitted')
+
+
+class SliceDhv(Device):
```
Comment (klauer) [inline pcdsdevices/piezo.py:88]: One minor thing - tab completion filtering and such won't be available on bare `ophyd.Device` classes. We generally mix in [`BaseInterface`](https://github.com/pcdshub/pcdsdevices/blob/8abc4868395ef923dba8bec68760823744cf0394/pcdsdevices/interface.py#L172) to get status printing helpers, tab filtering, etc.

If this is solely for typhos-based usage, it's probably OK to skip that for now.

Comment (ZryletTC) [reply pcdsdevices/piezo.py:88]: Does it hurt to add BaseInterface? If not, might as well just in case we end up using them in hutch-python at some point.

Comment (slactjohnson) [reply pcdsdevices/piezo.py:88]: I will certainly want to use this in hutch python (part of why I wrote this :) ). I'll add that in.

**Comment by ZLLentz** [comment]
All comments addressed, checklist complete, merging!

**Pull request merged**